### PR TITLE
Target Sum triplets in Python

### PR DESCRIPTION
**Pull request by shubham5351**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description
Program in Python to print all triplets of numbers which sum to target.
For example
input:
9
5 7 9 1 2 4 6 8 3
10
output:
1, 2 and 7
1, 3 and 6
1, 4 and 5
2, 3 and 5

### Checklist

- [✓] I've read the contribution guidelines.
- [✓] I've checked the issue list before deciding what to submit.
- [✓] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Complexity Analysis-
Time Complexity - O(n^2)
Space Complexity - O(1)

### CODE REVIEW

File: Python/cp/Target_Sum_Triplet.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        current = lst[i]
+        while left < right:
+            if current + lst[left] + lst[right] == target: # Checking for the sum if it's equal to target
+                print(current, ' ',lst[left], ' ', lst[right])
+                left += 1
+                right -= 1
+            elif current + lst[left] + lst[right] < target: # Checking sum is less than target if it is then increment left
+                left += 1
+            else: # Otherwise decrement right
+                right -= 1
```
Comment (atarax665) [inline Python/cp/Target_Sum_Triplet.py:18]: The function is incomplete. It ain't returning anything.

File: Python/cp/Target_Sum_Triplet.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+                right -= 1  
+                
+lst = []
+rang = int(input('Enter the Range for the Triplet:'))  
+for i in range(rang):
+    inp = int(input()) # Taking the input from the user
+    lst.append(inp) # Appending the input elements at the end of the list
+op = int(input('Enter the target for triplet:'))
+print('The Target Triplet is')
+targettriplets(lst, len(lst), op)
```
Comment (atarax665) [inline Python/cp/Target_Sum_Triplet.py:27]: Add sample input/output in commented form.

File: Python/cp/Target_Sum_Triplet.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+def targettriplets(lst, rang, target):
```
Comment (atarax665) [inline Python/cp/Target_Sum_Triplet.py:3]: Add description of the problem in the beginning with time and space complexity of the algorithm you use.

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Comment by shubham5351** [comment]
Ok I will do the changes

File: Python/cp/Target_Sum_Triplet.py (hunk 1 above)
Comment (shubham5351) [reply Python/cp/Target_Sum_Triplet.py:18]: I am not returning any value as the condition satisfies I am just printing the values.The function is also complete you can review it

File: Python/cp/Target_Sum_Triplet.py (hunk 2 above)
Comment (shubham5351) [reply Python/cp/Target_Sum_Triplet.py:27]: I have also added the input/output in commented form.Please Review it.

**Comment by shubham5351** [comment]
Can any one tell me why there is this branch conflict I am not able to clarify it.

**Pull request merged**